assets/js/script.js

```diff
@@ -143,4 +143,21 @@ const showModal = () => {
     });
   });
 };
-showModal();
\ No newline at end of file
+showModal();
+
+const error = document.querySelector('.errorMsg');
+const submitBtn = document.querySelector('#submit-btn');
+const form = document.querySelector('.contact-form');
+const email = document.querySelector('#email');
+const emailInput = email.value;
+
+submitBtn.addEventListener('click', (e) => {
+  const regEx = /^([a-z0-9_\-.]+)@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,3})$/;
+
+  if (emailInput.match(regEx)) {
+    form.submit();
+  } else {
+    e.preventDefault();
+    error.innerHTML = 'Please enter a valid email adress in lowercase letters only';
+  }
+});
```

Comment:
- [ ] As in the requirements, if a user submits the contact form with the email field containing only lowercase characters the form is sent normally but that is not the case for you, even if the email field contains only lowercase characters the form didn't send normally. Please kindly make sure your form works correctly 🙃 😉 

<img width="771" alt="Screen Shot 2022-09-07 at 23 21 36" src="https://user-images.githubusercontent.com/74501523/188984785-c05c6a70-88a5-40e8-b6b0-84f9e8ff3cb4.png">